app/org/sagebionetworks/bridge/play/controllers/FPHSController.java

```diff
@@ -0,0 +1,63 @@
+package org.sagebionetworks.bridge.play.controllers;
+
+import static org.sagebionetworks.bridge.Roles.ADMIN;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.models.accounts.ExternalIdentifier;
+import org.sagebionetworks.bridge.models.accounts.FPHSExternalIdentifier;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.services.FPHSService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import play.mvc.Result;
+
+@Controller("fphsController")
+public class FPHSController extends BaseController {
+    
+    private static final TypeReference<ArrayList<FPHSExternalIdentifier>> externalIdTypeRef = 
+            new TypeReference<ArrayList<FPHSExternalIdentifier>>() {};
+
+    private static final BridgeObjectMapper mapper = BridgeObjectMapper.get();
```

Comment:
Changed to all caps. Changed ArrayList to List (it might be the case that Jackson needs a concrete implementation, we'll see).